src/invidious/channels.cr

```diff
@@ -938,34 +939,18 @@ def get_about_info(ucid, locale)
 end
 
 def get_channel_videos_response(ucid, page = 1, auto_generated = nil, sort_by = "newest", youtubei_browse = true)
```

Comment:
The `youtubei_browse` parameter can be removed.

Reply:
Yes, forgot to. Fixed in 3e88b72.